{"uid":"16a89c0b4b9ea3cb","name":"SyncManager sync throws error when documents sync fails","historyId":".frontend.packages.internal.catalyst_voices_services.test.src.sync.sync_manager:.frontend.packages.internal.catalyst_voices_services.test.src.sync.sync_manager#SyncManager sync throws error when documents sync fails","time":{"start":1746611523000,"stop":1746611523041,"duration":41},"status":"passed","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"testStage":{"steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false},"afterStages":[],"labels":[{"name":"resultFormat","value":"junit"},{"name":"suite","value":".frontend.packages.internal.catalyst_voices_services.test.src.sync.sync_manager"},{"name":"testClass","value":".frontend.packages.internal.catalyst_voices_services.test.src.sync.sync_manager"},{"name":"package","value":".frontend.packages.internal.catalyst_voices_services.test.src.sync.sync_manager"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"history":{"statistic":{"failed":0,"broken":0,"skipped":0,"passed":2,"unknown":0,"total":2},"items":[{"uid":"d4bdaae56048b7fc","reportUrl":"https://input-output-hk.github.io/catalyst-voices/allure-action/fix_clearing_registration_progress_2474/test-report/14879657424_1746611116786/#testresult/d4bdaae56048b7fc","status":"passed","time":{"start":1746609951000,"stop":1746609951036,"duration":36}}]},"tags":[]},"source":"16a89c0b4b9ea3cb.json","parameterValues":[]}